src/plugins/irc/irc-ctcp.c

```diff
@@ -862,7 +932,9 @@ irc_ctcp_recv_dcc (struct t_irc_server *server, const char *nick,
                 weechat_infolist_new_var_string (item, "proxy",
                                                  IRC_SERVER_OPTION_STRING(server, IRC_SERVER_OPTION_PROXY));
                 weechat_infolist_new_var_string (item, "remote_address", pos_addr);
+                weechat_infolist_new_var_string (item, "local_address", str_address);
                 weechat_infolist_new_var_integer (item, "port", atoi (pos_port));
+                weechat_infolist_new_var_string (item, "token", pos_token);
```

Comment:
`token` field should be an integer here:

```C
                weechat_infolist_new_var_integer (item, "token", atoi (pos_token));
```

Reply:
Done.